### ISSUE TITLE
Works perfectly, but..

### ISSUE THREAD

**Issue by nudam00** [masihsultani/StockXCrawler#1]
It works perfectly but there is one problem, it only scraps 250 sales cuz there is no more shown on the page. Do you have the same problem?

**Comment by masihsultani** [comment]
Yes sadly stockx has been making it more difficult to scrap their data, I believe they are now limiting it to 250 transactions.
